packages/gp2-components/src/molecules/EventSpeakers.tsx

```diff
@@ -68,7 +72,7 @@ type UserSpeakerProps = {
 const UserSpeaker = ({ speaker }: UserSpeakerProps) => (
   <Link
     href={gp2Routing.users({}).user({ userId: speaker.id }).$}
-    overrideStyles={css({ display: 'flex' })}
+    overrideStyles={css([{ display: 'flex' }, textWrap])}
```

Comment:
In an effort to reduce the overrideStyles, can we move this styles to a span within the link?